auth/views.py

```diff
@@ -10,11 +11,25 @@ def bango(request):
 
 
 def braintree(request):
+    """
+    Pass the request through to Braintree. There are two jobs to do:
+    1) Add in the Braintree auth into the HTTP headers
+    2) Ensure that requests will check the correct Braintree crt.
+    """
     new_request = prepare(request)
+
+    # Add in the correct Braintree Authorization.
     new_request['headers']['Authorization'] = b"Basic " + encodebytes(
         settings.BRAINTREE_PUBLIC_KEY.encode('ascii') + b":" +
         settings.BRAINTREE_PRIVATE_KEY.encode('ascii')).strip()
-    return send('braintree', new_request)
+    # Taken from http://bit.ly/1cBESdC and ensures that the
+    # crt is passed through to the requests verify parameter.
+    new_request['verify'] = (
+        '{url.scheme}://{url.netloc}{crt}'.
```

Comment:
this is really confusing to me. Why would you set a cert source to a URL? How would you verify the cert for the URL to get the cert? Also, the docs in requests seem to suggest that this needs to be a path to a local file on disk.

The code you linked to seems to load a local cert file, not specify a URL (unless I'm looking at the wrong thing). What it does is use [inspect.getfile(Environment)](https://github.com/braintree/braintree_python/blob/master/braintree/environment.py#L47) to look up the path on disk (within the braintree python module) to the distributed SSL cert file. For example: https://github.com/braintree/braintree_python/tree/master/braintree/ssl Isn't that what we want to do?